lib/t_web/channels/feed_channel.ex

```diff
@@ -32,11 +34,18 @@ defmodule TWeb.FeedChannel do
         |> Matches.list_matches()
         |> render_matches(screen_width)
 
+      expired_matches =
+        user_id
+        |> Matches.list_expired_matches()
+        |> render_expired_matches(screen_width)
+
       reply =
         %{}
+        |> Map.put("match_expiration_duration", @match_expiration_duration)
```

Comment:
needed for client to always have an up-to-date match_expiration_duration which enable to correctly show % of time left till expiration (i.e. 10 / 48 hours = circle with 21% filled)